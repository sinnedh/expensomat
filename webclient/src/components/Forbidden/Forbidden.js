import React from 'react'

const Forbidden = () => (
  <p>You don't have the credentials to view this page</p>
)

export default Forbidden
