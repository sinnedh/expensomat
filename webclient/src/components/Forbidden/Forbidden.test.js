import React from 'react'
import ReactDOM from 'react-dom'
import Forbidden from './Forbidden'

describe('Forbidden', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Forbidden />)
    expect(wrapper).toMatchSnapshot()
  })
})
