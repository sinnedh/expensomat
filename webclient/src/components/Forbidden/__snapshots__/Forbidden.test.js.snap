// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forbidden matches snapshot 1`] = `
<p>
  You don't have the credentials to view this page
</p>
`;
